pydent/aqsession.py

```diff
@@ -25,10 +25,9 @@
     first10 = session.SampleType.first(10)
     mysamples = session.Sample.last(10, query={'user_id': 66})
 
-.. seealso::
```

Comment:
should be `seealso`, this is a sphinx directive
https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html